Common.Configurations.targets

```diff
@@ -33,7 +33,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.21" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.35" />
+	<PackageReference Include="LT.DigitalOffice.Models" Version="1.0.31" />
```

Comment:
remove extra tab

Reply:
and please use kernel 37, version 35 is not correct

Reply:
done